### PR TITLE
Dashboard graphs made

### PR DESCRIPTION
**Pull request by lukieh2014**
Dashboard.py script completed, with all graphs working and tested locally. Referring to ticket #68 .

### CODE REVIEW

File: pipeline/load.py
Comment (GabrielNsiah) [inline pipeline/load.py:1]: Could you add some type hints to the functions?

File: pipeline/load.py [hunk 1]
```diff
@@ -80,15 +80,18 @@ def find_botanist_id(cursor, first_name, last_name, email, phone) -> int:
 
 def find_plant_id(cursor, plant_name: str,
                   scientific_name_id: int, location_id: int, image_url: str) -> int:
-    """Returns the botanist as shown in the database based on data given."""
```
Comment (GabrielNsiah) [inline pipeline/load.py:83]: Good catch fixing the duplicate docstrings.

File: streamlit/continents.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+    all_data["recording_taken"] = pd.to_datetime(all_data["recording_taken"])
+
+    all_data["recording_taken"] = all_data["recording_taken"].dt.floor("H")
+
+    grouped_df = all_data.groupby(["recording_taken", "continent_name"]).agg(
+        avg_soil_moisture=("soil_moisture", "mean")
+    ).reset_index()
+
+    print("Making Graph")
```
Comment (GabrielNsiah) [inline streamlit/continents.py:69]: Does this print statement need to be here?

File: streamlit/dashboard.py [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+        .rename(columns={"recording_taken": "last_watered"})
+    )
+    return last_watered
+
+
+def prepare_data_for_chart(df):
+    """
+    Melt the dataframe for chart compatibility.
+    """
+    return df.melt(id_vars="minute", var_name="Measurement", value_name="Value")
```
Comment (GabrielNsiah) [inline streamlit/dashboard.py:57]: Would it be more readable to have a variable than you then return?

**Review by RidwanHamid501 — APPROVED** [review]
Nice!

**Pull request merged**